sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_models.py

```diff
@@ -40,45 +37,28 @@ def _from_generated(cls, permissions):
 
 
 class KeyVaultRoleAssignment(object):
-    """Represents the assignment to a principal of a role over a scope"""
+    """Represents the assignment to a principal of a role over a scope
+
+    :ivar str name: the assignment's name
+    :ivar KeyVaultRoleAssignmentProperties properties: the assignment's properties
+    :ivar str role_assignment_id: unique identifier for the assignment
+    :ivar str type: type of the assignment
+    """
 
     def __init__(self, **kwargs):
         # type: (**Any) -> None
-        self._role_assignment_id = kwargs.get("role_assignment_id")
-        self._name = kwargs.get("name")
-        self._properties = kwargs.get("properties")
-        self._type = kwargs.get("assignment_type")
+        self.name = kwargs.get("name")
```

Comment:
Is there a particular reason to move away from properties?

Reply:
Architect opinion: https://github.com/Azure/azure-sdk/blob/master/docs/python/_includes/example_client.py#L26